tests/suites/spaces_ec2/juju_bind.sh

```diff
@@ -22,7 +22,8 @@ run_juju_bind() {
 	configure_multi_nic_netplan "$juju_machine_id" "$hotplug_iface"
 
 	# Deploy test charm to dual-nic machine
-	juju deploy ./testcharms/charms/space-defender --bind "defend-a=alpha defend-b=isolated" --to "${juju_machine_id}"
+	# shellcheck disable=SC2046
```

Comment:
What happens with the output of pack_charm here if it fails for some reason? Does doing it in line like this hide the error in any way? If it does we should do it as a separate line cuz it'll make the tests a lot easier to debug

Reply:
I'm not sure, can you test it?